### PR TITLE
Web strings reworked 5

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by mitchdowney** [comment]
@comradekingu there are too many changes in this for me to comment at this time. Anything that involves changes to "free" or "open source" we need to discuss. The rule of thumb is, if it is description text, we will generally go with your "copylefted libre" related changes. If it is marketing text like "open source", we're sticking with Open Source for US marketing.

If you want changes to "Open Source" or "Free" or those terminology, please create a Github issue for those discussions if you want to have them.

Separately from that...the keys that begin with:

`category -`
`filter sorting -` 

Are tied to some hardcoded logic in our apps, and enabling translations without & or with – instead of - will need their own separate issues to fix the hardcoded logic.

There are changes in here that I'd like to merge in, but there's too much stuff that is a _philosophical debate about the political significance of language_ and not a straight-forward base language improvement. I'd appreciate if you separate PRs like this into base language improvements and philosophical improvements that would help speed up PR merges and help focus discussion.

**Comment by mitchdowney** [comment]
@comradekingu actually, I am wrong about the category and filter translations. I don't have the keys tied to translations, so the "and" and "–" character changes will be fine. No Github issues needed.

**Comment by comradekingu** [comment]
@mitchdowney
I tried going over it to make amends, but it just doesn't make any sense to not explain things right there and then in the strings.

"Open Source" purposefully does not focus on freedoms.
Meanwhile, it seems to me Podverse is trying to sell ideas of granted freedoms for reasons why they are important.
The audience isn't even the old business world that was at one time scared by it…
The mismatch is complete, and there is no searchability there.

What I am trying to do is to arrive at a communicated understanding of real developers behind it, instead of something that is just free as in how open source is (mis)understood.
And what a surprise nobody understands purposefully muddy language.
Ask 100 regular people what "Open Source" is, and you will most often get ~"gratis".
The freedoms are undercommunicated for business, and similarly not understood by users.
Better marketing in terms of how RedHat made their money, sure. That however doesn't carry over at all.

I am sorry to say, but "All Podverse software is provided under a free and open source (FOSS) licence."
doesn't communicate anything to anyone. People are either in the know, or none the wiser.
There are no other categories.

If the user doesn't already know what kinds of terms are licensed in software, there is no telling what "open source" is.
Moreover, "free" here makes the most legwork possible trying to qualify "open source".
If it is to mean the same, _and_ be fully redundant at that (outside of OSI and FSF differences in qualifying licenses),
who does that?
Pleonasms that don't reassure anyone doesn't make for functional language.
"FLOSS" is a similar "what if we add words" idea.

TL;DR
From my point of view Podverse making _marketing decisions_ is a commensurate disaster.
Between "we could do with some marketing" and "we don't know how to do marketing", there is a lot of "do nothing" that works better.

**Comment by mitchdowney** [comment]
Sorry @comradekingu, I greatly appreciate your help, and it pains me that my obstinance on this could discourage you, but this PR is going to stay in limbo as long as there are changes to our "open source" and "free and open source" marketing language. As I mentioned elsewhere...at this time we're accepting any "libre" and "copylefted libre" that are not part of our marketing language. Some of your changes in this PR would fall under that criteria, and some would not. We're way over our head right now with urgent server work and bug fixes, and it's hard for me to prioritize the changes here and potentially have a philosophical / marketing debate.

That said, to keep this from staying completely in limbo, I'll plan on branching-off your PR and reverting a few parts of it myself. I just would really prefer if you keep the philosophical changes (basically anything that changes "open source") to their own PR so the discussion can be isolated there. Most of these changes here are not philosophical but rather improve the clarity and ease for translations. Those I could do a quick review of and merge in immediately. Changes to "open source" though would need both @kreonjr and I to agree to it.

To be honest I am *warming* to the idea of "libre" and "copylefted libre" more than I was at the beginning of this year. I at minimum however think open source should be included for SEO purposes. It's not "wrong" imo to say we create open source software. It just doesn't clarify that all our software is also FLOSS.

If you use Matrix and ever want to chat with us in real-time we have a "Translations" room you could join: https://matrix.to/#/#podverse-space:matrix.org

File: public/locales/en/common.json [hunk 1]
```diff
@@ -61,15 +61,15 @@
...
   "confirmed": "confirmed",
   "Contact": "Contact",
   "ContactSupport": "If it still doesn't appear, please e-mail <1>contact@podverse.fm</1> for help.",
   "ContactXMPPChatRooms": "XMPP chat-rooms",
-  "ContactXMPPText1": "XMPP is free and open technology letting anyone create and host their own universal chat platform.",
+  "ContactXMPPText1": "XMPP is libre technology letting anyone create and host their own universal chat platform.",
   "ContactXMPPText2": "If you have an XMPP account, you can log in using our official chat client:",
   "Contribute": "Contribute",
-  "Contribute info": "If you are interested in helping Podverse in any capacity, please reach out to us using one of the options on our Contact page.",
+  "Contribute info": "Help Podverse in any capacity by please reach out on the contact page.",
```
Comment (comradekingu) [inline public/locales/en/common.json:72]:
```suggestion
  "Contribute info": "Help Podverse in any capacity by reaching out on the contact page.",
```

**Comment by comradekingu** [comment]
@mitchdowney
I can only permeate you in logic in the hopes it soaks in.
#1028

https://remmina.org/fastforward/ is a shamelessly self-plugging venture that stops all attempts at marketing.
Short of the masses putting their money where my mouth ends, I will pay you for those changes.

And hark! how blithe the throstle sings!

**Comment by mitchdowney** [comment]
> @mitchdowney I can only permeate you in logic in the hopes it soaks in. #1028 Short of the masses putting their money where my mouth ends, I will pay you for those changes.
> 
> Wrt. the above, https://remmina.org/fastforward/ is a shamelessly self-plugging venture that stops all attempts at marketing.
> 
> And hark! how blithe the throstle sings!

Very interesting. I know next to nothing about organizations that share our copyleft values. I'm either blocked by work or mentally blocked from reaching out and figuring out ways to collaborate with them. If you'd ever like to reach out to them please feel free and I'd be happy to talk with anyone interested in what we're doing.
